lib/screen/login_page.dart

```diff
@@ -0,0 +1,152 @@
+import 'package:flutter/material.dart';
+
+import '../utils/constants.dart';
+
+class LoginPage extends StatefulWidget {
+  const LoginPage({Key? key}) : super(key: key);
+
+  @override
+  State<LoginPage> createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: CustomPaint(
+        painter: Bluepainter(),
+        child: Container(
+          width: double.infinity,
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              // Sochem icon
+              Container(
+                width: MediaQuery.of(context).size.width * 0.5,
+                height: MediaQuery.of(context).size.height * 0.6,
+                decoration: BoxDecoration(
+                  shape: BoxShape.circle,
+                  image: DecorationImage(
+                    image: AssetImage(SochemIcon),
+                    fit: BoxFit.scaleDown,
+                  ),
+                ),
+              ),
+
+              //Buttons for login options
+              Container(
+                padding: EdgeInsets.symmetric(
+                    vertical: MediaQuery.of(context).size.height * 0.1),
+                child: Column(
+                  children: [
+                    Padding(
+                      padding: const EdgeInsets.only(bottom: 25),
+                      child:
+
+                          // Login button
+                          ElevatedButton(
+                        onPressed: () {},
+                        child: Text(
+                          "LOGIN",
+                          style: TextStyle(color: Colors.blue),
+                        ),
```

Comment:
This button should signify that it is a Google Login. Also introduce the logic for Google Sign In using the neccessary API.